### PR TITLE
Added CircleCi Status

### PR DESCRIPTION
**Pull request by drazisil**
Please provide the following information:
- Link to your code repository: https://github.com/drazisil/sublime-circleci/
- Link to the tags page with at least one [semver](http://semver.org) tag:  https://github.com/drazisil/sublime-circleci/releases/tag/v.0.5.0-beta

Also make sure you:
1. Used `"tags": true` and not `"branch": "master"` ([versioning docs](https://packagecontrol.io/docs/submitting_a_package#Step_4)) Yes
2. Ran the tests ([tests docs](https://packagecontrol.io/docs/submitting_a_package#Step_7)) Yes - Pass

### CODE REVIEW

**Comment by joshua-golub** [comment]
Drazisil-

I am reviewing this package but have no idea what it actually **does**. Would you please update the `README.md` in your repository to provide a useful explanation?

@FichteFoll, this plugin is using a version number of `v.0.5.0-beta`. I know that `v0.5.0-beta` is valid, but is it also valid to have a period (`.`) after the `v`?

-Josh

**Comment by FichteFoll** [comment]
@joshua-golub, technically any prefix is accepted, but it needs to be specified in the `"tags"` field of the release. By default, PC only recognizes no prefix and `v`.

@drazisil, as mentioned above, please add a tag that does not have a dot after `v`.
Furthermore, note that, by default, pre-release versions are not installed by Package Control. This has to be enabled in its settings before attemption to install your package. I suggest releasing an non-prerelease version.

**Comment by drazisil** [comment]
README updated and also tagged a version v1.0.0

**Comment by FichteFoll** [comment]
Due to `self.view.file_name().rpartition('/')[0]`, your package won't work on Windows. It will if you use `os.path.dirname(self.view.fil_name())` however.

**Comment by drazisil** [comment]
@FichteFoll I didn't mark windows because I couldn't test on that. If thats all it takes though. What about https://github.com/drazisil/sublime-circleci/blob/master/CircleCi.py#L43?

**Comment by FichteFoll** [comment]
>What about https://github.com/drazisil/sublime-circleci/blob/master/CircleCi.py#L43?

That operates on a URI, no? There are no differences for platforms in that regard.

>Also, self.view.fil_name() or self.view.file_name() ?

Ah, `file_name` of course.

**Comment by drazisil** [comment]
Done

File: repository/c.json [hunk 1]
```diff
@@ -935,6 +935,17 @@
 			]
 		},
 		{
+			"name": "CircleCi Status",
+			"details": "https://github.com/drazisil/sublime-circleci/",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master",
```
Comment (FichteFoll) [inline repository/c.json]: You must remove this line

**Pull request merged**